test/mi.c

```diff
@@ -753,6 +753,13 @@ static void test_resp_csi(nvme_mi_ep_t ep)
 
 	rc = nvme_mi_mi_read_mi_data_subsys(ep, &ss_info);
 	assert(rc != 0);
+
+	nvme_mi_set_csi(ep, 1);//Change CSI
+
+	rc = nvme_mi_mi_read_mi_data_subsys(ep, &ss_info);
+	assert(rc != 0);
```

Comment:
Should we also add a test that the CSI is received as expected (ie, in the callback)?